adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -29,7 +28,9 @@ class AkkaHttpAdapter private (private val options: AkkaHttpServerOptions)(impli
     interpreter: HttpInterpreter[R, E]
   )(implicit runtime: Runtime[R], materializer: Materializer): Route =
     akkaInterpreter.toRoute(
-      interpreter.serverEndpoints[R].map(convertHttpStreamingEndpoint[R, (GraphQLRequest, ServerRequest)])
+      interpreter
+        .serverEndpoints[R, AkkaStreams](AkkaStreams)
```

Comment:
I could probably go further an introduce a partially applied method here, but I don't think it is necessary since this really should only be used by people who know what they are doing